recipes/core/busybox/inittab

```diff
@@ -0,0 +1,25 @@
+# Startup the system
+::sysinit:/bin/mount -t proc proc /proc
+::sysinit:/bin/mount -o remount,rw /
+# The linux kernel does not mount /dev in case of an ramdisk. So check if it is
+# already mounted and if not mount it before creating any subdirs in it.
+::sysinit:/bin/sh -c '/bin/grep -q /dev /proc/mounts || mount /dev'
+::sysinit:/bin/mkdir -p /dev/pts /dev/shm
+::sysinit:/bin/mount -a
+::sysinit:/bin/mkdir -p /run/lock/subsys
+::sysinit:/sbin/swapon -a
+null::sysinit:/bin/ln -sf /proc/self/fd /dev/fd
+null::sysinit:/bin/ln -sf /proc/self/fd/0 /dev/stdin
+null::sysinit:/bin/ln -sf /proc/self/fd/1 /dev/stdout
+null::sysinit:/bin/ln -sf /proc/self/fd/2 /dev/stderr
```

Comment:
I don't get this. `/proc/self` is a magic symlink. It will point to the PID of the process. In this case each line will have a different PID, creating the symlink in a different directory. As we mounted proc before, these should exist already AFAIK. Could you shed some light on why this is needed here?